backend/app/api/v1/commons/utils.py

```diff
@@ -39,7 +39,9 @@ def isRehearse(job):
 
 
 def clasifyAWSJobs(job):
-    if job["upstreamJob"].__contains__("rosa-hcp"):
+    if ("rosa-hcp" in job["clusterType"]) or ("rosa" in job["clusterType"]
```

Comment:
Thanks! That looks a lot nicer, and it's more compact as well.

I don't think `black` will like the hanging continuation lines, but since we've only started applying `black` on the `revamp` branch I'm explicitly not mentioning that here. 😆

Reply:
I think once we get `revamp` changes merged which has black, Then we can raise a separate PR which corrects formatting for the entire repo.

Reply:
Yeah, that was what Varshini and I were thinking, so I'm glad we align. I wouldn't want to do massive refactoring the the number of branches we have outstanding now -- the rebases will be nasty. But a single reformatting PR after we're stable may be big, but it should be easy to review.